prettier.config.js

```diff
@@ -0,0 +1,8 @@
+import config from '@vercel/style-guide/prettier';
+
+export default {
+  ...config,
+  jsxSingleQuote: true,
+  trailingComma: 'es5',
+  plugins: ['prettier-plugin-tailwindcss'],
```

Comment:
you should add previous plugins from vercel
```suggestion
  plugins: [ ...config.plugins, 'prettier-plugin-tailwindcss'],
```